app/Auth/CustomGate.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Rogue\Auth;
+
+use Illuminate\Support\Arr;
+use Illuminate\Auth\Access\Gate;
+use Illuminate\Contracts\Auth\Access\Gate as GateContract;
+
+class CustomGate extends Gate implements GateContract
+{
+    protected function raw($ability, $arguments = [])
```

Comment:
It'd be nice to add a docblock explaining what we've overridden here:

```
    /**
     * Get the raw result from the authorization callback. We've overridden this
     * method to allow users to be `null` so policies can apply to anonymous users.
     *
     * @param  string  $ability
     * @param  array|mixed  $arguments
     * @return mixed
     */
```

Reply:
ah great call!